packages/@divvi/mobile/src/app/App.tsx

```diff
@@ -89,15 +89,21 @@ export class App extends React.Component<Props> {
 
   render() {
     return (
-      <SafeAreaProvider>
+      <SafeAreaProvider
+        style={Platform.OS === 'android' ? { marginTop: initialWindowMetrics?.insets.top } : {}}
+      >
```

Comment:
Curious, what about keeping this explicit margin and still having edge-to-edge enabled?